documentation/migration-guides/Az.12.0.0.0-migration-guide.md

```diff
@@ -0,0 +1,2762 @@
+    - The change is expected to take effect from version : '4.0.0'
+
+#### Before
+```powershell
+(Status is of type EntityStatus)
+```
+#### After
+```powershell
+(Status is of type String)
+```
+
+
+## Az.Sql
+
+### `New-AzSqlDatabaseFailoverGroup`
+
+- Cmdlet breaking-change will happen to all parameter sets
+  - The default value of FailoverPolicy will change from Automatic to Manual
+  - This change is expected to take effect from Az.Sql version: 5.0.0 and Az version: 12.0.0
+
+#### Before
+```powershell
+New-AzSqlDatabaseFailoverGroup -ServerName example-primary-server -ResourceGroupName example-rg -PartnerServerName example-secondary-server -FailoverGroupName example-fg     (FailoverPolicy defaults to Automatic)
+```
+#### After
+```powershell
+New-AzSqlDatabaseFailoverGroup -ServerName example-primary-server -ResourceGroupName example-rg -PartnerServerName example-secondary-server -FailoverGroupName example-fg     (FailoverPolicy defaults to Manual)
+```
+
+
+### `Set-AzSqlDatabaseFailoverGroup`
+
+- Cmdlet breaking-change will happen to all parameter sets
+  - The default value of FailoverPolicy will change from Automatic to Manual
+  - This change is expected to take effect from Az.Sql version: 5.0.0 and Az version: 12.0.0
+
+#### Before
+```powershell
+Get-Help Set-AzSqlDatabaseFailoverGroup -Parameter FailoverPolicy (Output shows default value = Automatic)
+```
+#### After
+```powershell
+Get-Help Set-AzSqlDatabaseFailoverGroup -Parameter FailoverPolicy (Output shows default value = Manual)
+```
+
+
+## Az.Storage
+
+### `Get-AzStorageQueue`
+
+- Cmdlet breaking-change will happen to all parameter sets
+  - The child property CloudQueue and EncodeMessage from deprecated v11 SDK will be removed. Use child property QueueClient instead of CloudQueue.
+  - This change is expected to take effect from Az.Storage version: 7.0.0 and Az version: 12.0.0
+
+#### Before
+```powershell
+PS C:\WINDOWS\system32> $queue = Get-AzStorageQueue -Context $ctx-Name myqueue
+PS C:\WINDOWS\system32> $queue | fl
+
+CloudQueue              : Microsoft.Azure.Storage.Queue.CloudQueue
+Uri                     : https://mystorageaccount.queue.core.windows.net/myqueue
+ApproximateMessageCount : 0
+EncodeMessage           : True
+QueueClient             : Azure.Storage.Queues.QueueClient
+QueueProperties         : Azure.Storage.Queues.Models.QueueProperties
+Context                 : Microsoft.WindowsAzure.Commands.Storage.AzureStorageContext
+Name                    : myqueue
+
+PS C:\WINDOWS\system32> $queueMessage = [Microsoft.Azure.Storage.Queue.CloudQueueMessage]::new("This is message 1")
+PS C:\WINDOWS\system32> $queue.CloudQueue.AddMessageAsync($queueMessage)
+```
+#### After
+```powershell
+PS C:\WINDOWS\system32> $queue = Get-AzStorageQueue -Context $ctx-Name myqueue
+PS C:\WINDOWS\system32> $queue | fl
+
+QueueClient             : Azure.Storage.Queues.QueueClient
+Uri                     : https://mysorageaccount.queue.core.windows.net/myqueue
```

Comment:
Just noticed I had a typo here -- It should be mystorageaccount instead of mysorageaccount. 
Is it still possible for us to fix it?